### ISSUE TITLE
Adjust plugin start automatically on App Start when set to Start Manually

### ISSUE THREAD

**Issue by R0b3e** [adjust/unity_sdk#134]
Hi,

we are trying to use adjust to track installs in our unity app. However the plugin always seems to start at appstart and in sandbox mode, even when the plugin is set to "StartManually" and no Adjust GameObject is present in the Scene.
How can the plugin start and where does it get it's data if no GameObject with the Adjust Plugin is in any scene?
With the plugin starting in sandbox mode, we are unable to track the installs of our production version.

When we try to initialize and start the plugin manually from code, it always says Adjust already initialized. It then switches to Production Mode but the install was already tracked in Sandbos mode at this time.

We are using Unity 5.6.4f1 and Unity 2017.3.1f1 with the Adjust Plugin v4.13.0.
Tested on Android 5.0.2.

Thanks for any help.

**Comment by 2beens** [comment]
Hi @R0b3e 

Our SDK will initialise itself only when `Adjust.start(...)` is called, and this can happen in two scenarios:
 - 1 - Adjust GameObject is at least in one of the scenes, and `start manually` is set to `false` (this leads to Adjust being initialised automatically in Adjust GameObject `Awake(...)` method)
- 2 - Adjust.start(...) is being called somewhere else in the code, which leads to Adjust SDK being initialised

Could you please make sure that you are calling `Adjust.start(...)` only once in your code? (And, when exactly are you calling it?)

**Comment by 2beens** [comment]
Hi @R0b3e

Our SDK will initialise itself only when `Adjust.start(...)` is called, and this can happen in two scenarios:
• 1 - Adjust GameObject is at least in one of the scenes, and `start manually` is set to `false` (this leads to Adjust being initialised automatically in Adjust GameObject `Awake(...)` method)
• 2 - Adjust.start(...) is being called somewhere else in the code, which leads to Adjust SDK being initialised

Could you please make sure that you are calling `Adjust.start(...)` only once in your code? (And, when exactly are you calling it?)

**Comment by uerceg** [comment]
@R0b3e 

Any update on this one?

**Comment by R0b3e** [comment]
Hi,

I removed the plugin several times and downloaded the latest version again.
Everything seems to be working fine now. 

Not sure was the error was though. Haven't chenged anything in my code.

sorry for late response.

**Comment by uerceg** [comment]
Thanks for the update! Feel free to reopen the issue if you encounter any similar problems.

Cheers!
